tests/test_answersfile_templating.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import pytest
+
+import copier
+
+from .helpers import build_file_tree
+
+
+@pytest.fixture(scope="module")
+def template_path(tmp_path_factory) -> str:
+    root = tmp_path_factory.mktemp("template")
+    build_file_tree(
+        {
+            root
+            / ".copier-answers-{{module_name}}.yml": """\
```

Comment:
Follow [the docs](https://copier.readthedocs.io/en/stable/configuring/#the-copier-answersyml-file):

```suggestion
            / "{{ _copier_conf.answers_file }}.jinja": """\
```